src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -23,10 +28,32 @@ const ConnectedChapterFilter = connect(
   })
 )(ChapterFilter);
 
+// tslint:disable-next-line: variable-name
+const StyledColorFilter = styled(ColorFilter)`
+  min-width: 29rem;
+  ${theme.breakpoints.mobileSmall(css`
+    /* 12.1 is a width of Chapter button */
```

Comment:
if this is subject to the font size then it is brittle, we should either set its width explicitly or calculate this somehow

Reply:
I had to change add `<FiltersTopBar>` and change some css stuff for Dropdown inside of it for smaller screens but it seems to work